sos/plugins/insights.py

```diff
@@ -13,15 +13,18 @@ class RedHatInsights(Plugin, RedHatPlugin):
     '''Collect config and log for Red Hat Insights
     '''
     plugin_name = 'insights'
-    packages = ['redhat-access-insights']
+    packages = ['redhat-access-insights', 'insights-client']
     profiles = ('system', 'sysmgmt')
-    conf_file = '/etc/redhat-access-insights/redhat-access-insights.conf'
+    conf_file = ('/etc/insights-client/insights-client.conf',
+                 '/etc/redhat-access-insights/redhat-access-insights.conf')
 
     def setup(self):
         self.add_copy_spec(self.conf_file)
         if self.get_option("all_logs"):
+            self.add_copy_spec("/var/log/insights-client/*.log*")
             self.add_copy_spec("/var/log/redhat-access-insights/*.log*")
         else:
+            self.add_copy_spec("/var/log/insights-client/insights-client.log")
             self.add_copy_spec("/var/log/redhat-access-insights/*.log")
```

Comment:
I had to keep the log file name as "/var/log/redhat-access-insights/*.log" because it was exceeding char limit and there is only one log file with extension ".log " old log files are named as *.log.1 , *.log.2 , *.log.3 ...etc

Reply:
We don't allow merge commits in `master` - please rebase your branch to only have the necessary commits for your changes and `git push --force`.